src/compiler/schema_interface.h

```diff
@@ -41,31 +41,31 @@ namespace grpc_generator {
 // Return formatted comments to be inserted in generated code.
```

Comment:
line 27-36.

Reply:
actually I think this is also showing in `src/compiler/config.h` and if we have `std::string` there, this block can just be removed.

Reply:
You're right. Thanks!